src/site/content/en/metrics/user-centric-metrics/index.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: User-centric metrics
```

Comment:
Another option might be to rename this `Performance metrics` (the directory name should change too if you rename the title)